src/Console/Command/DescribeCommand.php

```diff
@@ -97,7 +97,7 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $name = $input->getArgument('name');
+        $name = rtrim($input->getArgument('name'), " \t,");
```

Comment:
What is the purpose of this change?

Reply:
basically to support this;
```
$ php-cs-fixer describe braces,

In DescribeCommand.php line 119:
                                                    
  Rule "braces," not found. Did you mean "braces"?  
                                                    

describe <name>
```

which happens a lot when you are copy-pasting from a list

Reply:
Meh, it still feels weird to silently fix user input this way, I would keep the current behavior.

Reply:
why, the trailing comma and trailing space can never be part of the fixer name by design, so why not ignore?

Reply:
Because of [Fail-fast principle](https://en.wikipedia.org/wiki/Fail-fast): when there is an error, we should report it immediatly instead of trying to hide it silently.

If we want to go this way, maybe there are a lot of other places in the code where we could add such workarounds. Also, this change requires a test. All of this means more maintenance work, not worth it at all IMO.

Reply:
I don't consider a trailing comma an error, like PHP supports it in array lists as well. I think here UX is more important than a theory

Reply:
Well, I do prefer when a tool tells me when I make a mistake instead of hiding it and letting me continue with my wrong usage.

Reply:
sure, I just want to support `name,` because I think it is valid usage